infra/charts/feast-feature-server/samples/instrumentation.yml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.metrics }}
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: feast-instrumentation
+spec:
+  exporter:
+    endpoint: http://{{ .Values.service.name }}-collector.{{ .Release.Namespace }}.svc.cluster.local:4318 # This is the default port for the OpenTelemetry Collector
```

Comment:
do helm value references make sense in these samples anymore? shouldn't it just be an example endpoint instead?